sympy/geometry/tests/test_polygon.py

```diff
@@ -1,7 +1,8 @@
 from sympy import Abs, Rational, Float, S, Symbol, symbols, cos, pi, sqrt, oo
 from sympy.functions.elementary.trigonometric import tan
-from sympy.geometry import (Circle, Ellipse, GeometryError, Point, Point2D, Polygon, Ray, RegularPolygon, Segment, Triangle, are_similar,
-                            convex_hull, intersection, Line)
+from sympy.geometry import (Circle, Ellipse, GeometryError, Point, Point2D, \
+                            Polygon, Ray, RegularPolygon, Segment, Triangle, \
```

Comment:
Backslash would not be necessary if under the parentheses

Reply:
Ok! I understand how to use the backslah now. Thanks a lot!

Reply:
Fixing `\` is not worth a rebuild. This can be fixed in some future PR.